src/timeseriesflattenerv2/feature_specs/timedelta.py

```diff
@@ -36,10 +36,10 @@ def __post_init__(self):
             init_df=self.init_frame.df.rename(
                 {self.init_frame.value_timestamp_col_name: self.output_name}
             ),
-            value_col_name=self.output_name,
             entity_id_col_name=self.init_frame.entity_id_col_name,
             value_timestamp_col_name=self.output_name,
         )
+        self.value_frame.value_col_names = [self.output_name]
```

Comment:
Ensure this is checked somewhere.

Reply:
The `value_timestamp_col_name` is renamed to this above (that value was checked), so it exists